### DISCUSSION TITLE
Allow new signups option not working correctly

### DISCUSSION THREAD

**Discussion by Koltsz** [dani-garcia/vaultwarden#4086] [Q&A] [answered]
Hi,

I have set SIGNUPS_ALLOWED=false but it doesnt seem to stop new signups

![image](https://github.com/dani-garcia/vaultwarden/assets/36265128/e9a93bb2-d5f0-4ba5-be49-2894942abff4)

is there any other env var in the same catagory that could be forcing it true, if that makes sense. This is what I have set:

```
- DOMAIN=***
- LOGIN_RATELIMIT_MAX_BURST=10
- LOGIN_RATELIMIT_SECONDS=60
- ADMIN_RATELIMIT_MAX_BURST=10
- ADMIN_RATELIMIT_SECONDS=60
- SENDS_ALLOWED=true
- EMERGENCY_ACCESS_ALLOWED=true
- WEB_VAULT_ENABLED=true
- SIGNUPS_ALLOWED=false
- SIGNUPS_VERIFY=true
- SIGNUPS_VERIFY_RESEND_TIME=3600
- SIGNUPS_VERIFY_RESEND_LIMIT=5
- SIGNUPS_DOMAINS_WHITELIST=gmail.com
```

This is what I see in the admin console:
![image](https://github.com/dani-garcia/vaultwarden/assets/36265128/1cc68e8d-4c8e-4ea3-9d79-cfd2f8dfc64b)

**Comment by stefan0xC** [comment, marked as answer]
See https://github.com/dani-garcia/vaultwarden/discussions/4058#discussioncomment-7519087

**Reply by Koltsz** [reply]
Thanks for the quick reply
